README.md

```diff
@@ -72,6 +72,29 @@ filed named `.gitkeep`. Finally, create individual locks by making an empty file
   retrying to acquire a lock or release a lock. The default is 10 seconds.
   Valid values: `60s`, `90m`, `1h`.
 
+* `prometheus_push_gateway`: *Optional.* URL to the prometheus push gateway that will
+  expose the following metrics for each execution of the `out` step:
+
+    * `resource_pool_execution_time`:
+      Total execution time of the current operation.
+      Labels: `pool, branch, operation`
+
+    * `resource_pool_num_available_locks`:
+      Number of available/unclaimed locks in the current pool.
+      Labels: `pool, branch`
+
+    * `resource_pool_num_claimed_locks`:
+      Number of claimed locks in the current pool.
+      Labels: `pool, branch`
+    
+    > Notes:
+    All metrics are computed after performing the operation.
+    There is no `uri` label, because prometheus cannot (yet) handle slashes (`/`)
```

Comment:
Is the "(yet)" world because handling slashes is a planned future feature of Prom ? If not, I would remove the "(yet)", to keep the text simple.

Also, please uppercase P in prometheus

Reply:
They mention in the go 'docstring' that it's not supported yet. This feature would be nice to have, hence the mention here.